k8s/0.23.0/local/vsecm-distroless-fips.yaml

```diff
@@ -499,6 +501,14 @@ spec:
               value: "spiffe://vsecm.com/workload/vsecm-safe/ns/vsecm-system/sa/vsecm-safe/n/"
             - name: VSECM_SAFE_TLS_PORT
               value: ":8443"
+            - name: VSECM_SENTINEL_INIT_COMMAND_PATH
```

Comment:
These environment variables were not provided, so their default values were being used. 
Provided them here to be explicit.